src/extensions/datadog/Dockerfile

```diff
@@ -4,10 +4,10 @@ FROM ${REPO}:${PHP_VERSION} AS ext
 
 # https://github.com/DataDog/dd-trace-php/releases
 RUN if ([ $(php -r "echo PHP_MAJOR_VERSION;") -ge "8" ] && [ $(php -r "echo PHP_MINOR_VERSION;") -gt 3 ]); then \
-    curl -sSL "https://github.com/DataDog/dd-trace-php/releases/download/1.5.1/datadog-setup.php" --output /tmp/datadog-setup.php \
+    curl -sSL "https://github.com/DataDog/dd-trace-php/releases/download/1.12.1/datadog-setup.php" --output /tmp/datadog-setup.php \
       && php /tmp/datadog-setup.php --php-bin all --install-dir /opt/share/datadog/ --enable-profiling; \
   else \
-    curl -sSL "https://github.com/DataDog/dd-trace-php/releases/download/1.4.0/datadog-setup.php" --output /tmp/datadog-setup.php \
+    curl -sSL "https://github.com/DataDog/dd-trace-php/releases/download/1.4.2/datadog-setup.php" --output /tmp/datadog-setup.php \
```

Comment:
@mariusjp weet jij zo waarom je deze erbij hebt gezet? Er lijken nu 2 installatie commando's te zijn, deze if statement (waarom 1.4?) en eentje die je nu niet ziet in de PR, maar die er onder staat.